### PR TITLE
Main Menu Changes

### PR DESCRIPTION
**Pull request by michael-grace**
Added advanced studio training, uploader training and comments

### CODE REVIEW

**Comment by michael-grace** [comment]
don't merge until recorder booking gets added in soon

File: src/Templates/MyRadio/getOnAir.twig [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
 <li {% if not (studio_demoed and paid and contract_signed) %} class="menu-checklist-disabled"{% endif %}>
   <a href="{% if not (studio_demoed and paid) %}javascript:{% else %}{{MyRadio.makeURL(config, 'Scheduler', 'myShows')}}{% endif %}"
      title="{% if not has_show %}You've passed all the stages of training. Now it's time to apply for your very first show! {%
 else
 %}One show a term not enough? Click here to apply for another show.{% endif %}">Apply for a Show</a>
 </li>
+
+<!--Advanced Studio Training-->
+<li class="{% if not studio_demoed %}menu-checklist-disabled{% endif %}">
+    <a href="https://docs.google.com/forms/d/e/1FAIpQLSdaCSzLu7cNDnkr0nVVFkSpzgBFor_coIotY1Wt1o9cH9UyyA/viewform" title="Want to find out about more of the amazing things you can do in the studio? Get Advanced Training here!">
```
Comment (mstratford) [inline src/Templates/MyRadio/getOnAir.twig]: Hey :) I believe you're missing the actual link text and closing a tag :)

File: src/Templates/MyRadio/getOnAir.twig [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+</li>
+
+<!--Uploader Training-->
+<li class="{% if not (studio_demoed and contract_signed)%}menu-checklist-disabled{% endif %}">
+    <a href="https://docs.google.com/forms/d/e/1FAIpQLSeIkVMShKhuwtn5cKJyvJd242WNVf2vtn7elW3-myFEKrIPig/viewform" title="Before you upload songs to our music library, you need to learn some rules about uploading. Click here to get some training.">Get Music Upload Trained</a>
```
Comment (mstratford) [inline src/Templates/MyRadio/getOnAir.twig]: And here

**Comment by markspolakovs** [comment]
Tested in MyRadio-dev, seems mostly okay

**Comment by michael-grace** [comment]
ok, we're all good now

**Pull request merged**